liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/AbstractLiquibaseMojo.java

```diff
@@ -312,20 +314,36 @@ public abstract class AbstractLiquibaseMojo extends AbstractMojo {
      */
     private File driverPropertiesFile;
 
-    private boolean hasProLicense;
-
     /**
-     * Specifies your Liquibase Pro license key.
+     * Specifies your Liquibase Pro license key. This has been deprecated in favor of using
+     * "liquibase.liquibaseLicenseKey", but this property will continue to be operational.
      *
      * @parameter property="liquibase.liquibaseProLicenseKey"
      */
     @PropertyElement
-    protected String liquibaseProLicenseKey;
+    @Deprecated
+    private String liquibaseProLicenseKey;
+
+    /**
+     * Specifies your Liquibase license key.
+     *
+     * @parameter property="liquibase.liquibaseLicenseKey"
```

Comment:
This property should be `liquibase.licenseKey`. The pro one was made wrong but no need to continue that problem.